src/main/java/org/sagebionetworks/web/client/widget/entity/tabs/AbstractTablesTab.java

```diff
@@ -95,6 +95,10 @@ public abstract class AbstractTablesTab implements TablesTabView.Presenter, Quer
 
 	protected abstract String getTabDisplayName();
 
+	protected abstract String getTabDescription();
```

Comment:
Expected this to be on all tabs (part of TablesTabView.Presenter), is that incorrect?